requirements.txt

```diff
@@ -4,3 +4,4 @@ south
 whoosh
 ipy
 python-ldap
+djangorestframework==2.4.1
```

Comment:
Understandeable, but still not a good practice for software that is going to released to users. For example jessie has 2.4.3 and not 2.4.1.  We want to allow the sysadmins to use system packages in order to take advantage of the security releases of both python packages and distributions and not pin them to a specific release.

That being said, in things like heroku where you don't really ship software to users, but the service itself, I would recommend pinning versions (actually heroku already does)